test/org/javarosa/core/model/utils/test/DateUtilsTests.java

```diff
@@ -331,6 +334,40 @@ private void testCycle(Date in) {
 		}
 
 	}
+
+	public void testFormatting() {
+	    class LangJanSun {
+            private LangJanSun(String language, String january, String sunday) {
+                this.language = language;
+                this.january = january;
+                this.sunday = sunday;
+            }
+
+            private String language;
+	        private String january;
+	        private String sunday;
+        }
+
+        LangJanSun langJanSuns[] = new LangJanSun[] {
```

Comment:
From the Java Tutorial:

> Enum Types
> 
> An enum type is a special data type that enables for a variable to be a set of predefined constants. The variable must be equal to one of the values that have been predefined for it. Common examples include compass directions (values of NORTH, SOUTH, EAST, and WEST) and the days of the week.

This is different. It’s a collection of test data, not an enumeration of all possible values of something.